### ISSUE TITLE
[15.0] mail_debrand > Empty "Auth Signup : Connexion Odoo" email template

### ISSUE THREAD

**Issue by gregory-moka** [OCA/social#1075]
## Module
`mail_debrand`  `15.0`

## Describe the bug
Once installed, the mail that is used to invite new users to the database is empty.

## To Reproduce
**Affected versions**:

1. Install `mal_debrand`
2. Create a new user, that send an email // previw the template from technical > email templates > Auth Signup : Connexion Odoo
3.  It seems that this lines are in cause : 
```html
Have a look at the <a href="https://www.odoo.com/page/tour?utm_source=db&amp;utm_medium=auth" style="color: #875A7B;">Odoo Tour</a> to discover the tool.
```

**Expected behavior**
The content has to be used in the sent mail
![image](https://user-images.githubusercontent.com/45974412/220678360-dfac77fb-06c1-44be-a6ff-3883dd211cca.png)

**Additional context**
![image](https://user-images.githubusercontent.com/45974412/220678043-c953135b-53da-41c7-ad94-d345374f215e.png)

**Comment by jeffreyouni** [comment]
I have the same problem, please help!

**Comment by mathben** [comment]
Check my fix #1215 
Work for Odoo 14, maybe will work for Odoo 15.
